src/components/button/button.stories.js

```diff
@@ -1,32 +1,105 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { storiesOf } from '@storybook/react';
 import { text, select, boolean } from '@storybook/addon-knobs';
 import { action } from '@storybook/addon-actions';
 import OptionsHelper from '../../utils/helpers/options-helper';
-import notes from './notes.md';
-import Button from './button';
+import notes from './documentation/notes.md';
+import Info from './documentation/Info';
+import Button from '.';
+
+const defaultKnobs = () => {
+  const size = select('size', OptionsHelper.sizesRestricted, OptionsHelper.sizesRestricted[1]);
+
+  return {
+    as: select('as', OptionsHelper.themesBinary, OptionsHelper.themesBinary[1]),
+    children: text('children', 'Example Button'),
+    disabled: boolean('disabled', false),
+    iconPosition: select('iconPosition', [...OptionsHelper.buttonIconPositions, ''], ''),
+    iconType: select('iconType', [...OptionsHelper.icons, ''], ''),
+    onClick: action('click'),
```

Comment:
If you use `onClick: (evt) => action('click')(evt)`, you can get the event info in the action too.

Reply:
Ah cool, changed it to that 😃